### DISCUSSION TITLE
Does MontePy support native TRransformations?

### DISCUSSION THREAD

**Discussion by SterlingButters** [idaholab/MontePy#358] [Q&A] [answered]
I.e. "TR cards"

**Comment by MicahGale** [comment, marked as answer]
Yes it does. They are implemented as [transform](https://idaholab.github.io/MontePy/api/montepy.data_inputs.transform.html#montepy.data_inputs.transform.Transform) objects. 

Transforms can be applied to both [surfaces](https://idaholab.github.io/MontePy/api/montepy.surfaces.surface.html#montepy.surfaces.surface.Surface.transform), and [a filled cell](https://idaholab.github.io/MontePy/api/montepy.surfaces.surface.html#montepy.surfaces.surface.Surface.transform), via a [fill object](https://idaholab.github.io/MontePy/api/montepy.data_inputs.fill.html#montepy.data_inputs.fill.Fill.transform).

Transformations through `trcl` isn't supported yet.

**Reply by SterlingButters** [reply]
I read through more of the issues and found the answer but thanks for getting back
